internal/fwserver/server_getproviderschema.go

```diff
@@ -31,15 +29,6 @@ type GetProviderSchemaResponse struct {
 func (s *Server) GetProviderSchema(ctx context.Context, req *GetProviderSchemaRequest, resp *GetProviderSchemaResponse) {
 	resp.ServerCapabilities = s.ServerCapabilities()
 
-	metadataReq := provider.MetadataRequest{}
```

Comment:
Should we call `s.ProviderTypeName()` to set `providerTypeName` in order to preserve original behaviour?